client/src/pages/FormPage.js

```diff
@@ -1,24 +1,237 @@
 import React from 'react';
-import Header from '../components/Header';
-import FormCard from '../components/FormCard';
-import Footer from '../components/Footer';
 import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import Input from '../components/Input';
+import Planet from '../icons/Planet';
+import Lock from '../icons/Lock';
+import PrivateConfirmationCard from '../components/PrivateConfirmationCard';
+import PublicConfirmationCard from '../components/PublicConfirmationCard';
+import QuestionCardsAdded from '../icons/QuestionCardsAdded';
+import { addQuestion } from '../api/questions';
+import {
+  linearGradientBoxShadow,
+  flexRowWrapCenter,
+  flexColumnCenter,
+  noBorderOutlineBGTextDeco
+} from '../styles/General';
 
-const Main = styled.main`
+const BigContainer = styled.div`
+  ${linearGradientBoxShadow};
+  width: 340px;
+  height: 380px;
+  border-radius: 25px 0px;
+`;
+
+const Form = styled.form`
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: space-evenly;
+  height: 100%;
+  color: ${props => props.theme.colors.text1};
+  padding: 0px 20px;
+`;
+
+const Nav = styled.div`
   display: flex;
-  justify-content: center;
-  align-items: center;
-  flex-flow: column nowrap;
+  margin: 0 10px 0 0;
+`;
+
+const NavButton = styled.button`
+  ${noBorderOutlineBGTextDeco};
+  border: 2px solid ${props => props.theme.colors.text1};
+  border-radius: 10px;
+  font-size: 1em;
+  font-weight: bold;
+  margin: 0 0 0 10px;
+  color: ${props => props.theme.colors.text1};
+  width: 50%;
+  flex-grow: 1;
+`;
+
+const Status = styled.div`
+  ${flexRowWrapCenter};
+  width: 100%;
+  justify-content: space-around;
+`;
+
+const QuestionsCounter = styled.div`
+  display: flex;
+  align-self: center;
+`;
+
+const Text = styled.div`
+  align-self: center;
+  font-size: 1.5rem;
+`;
+
+const ErrorContainer = styled.div`
+  position: absolute;
+  z-index: 1000;
+  text-align: center;
+  color: ${props => props.theme.colors.warn};
+`;
+
+const TextWrapperOutsideCard = styled.div`
+  margin: 0 0 10px;
+  color: ${props => props.theme.colors.card2};
+`;
+
+const Main = styled.main`
+  ${flexColumnCenter};
   height: 100vh;
   width: 100vw;
 `;
 
-export default function FormPage() {
+export default function FormPage({ privateCode }) {
+  const [question, setQuestion] = React.useState({
+    question: '',
+    correct_answer: '',
+    incorrect_answer1: '',
+    incorrect_answer2: '',
+    incorrect_answer3: '',
+    privateCode: privateCode
+  });
+  const [questionStatus, setQuestionStatus] = React.useState('inactive');
+  const [questionsCounter, setQuestionsCounter] = React.useState(0);
+  const [finished, setFinished] = React.useState(false);
+  const [showError, setShowError] = React.useState(false);
+  console.log(privateCode);
```

Comment:
🌲save the forest, less logging!

Reply:
Hehe, will make a habit of searching for those climate killers next time before push ;)